java/rocksjni/jnicallback.cc

```diff
@@ -49,5 +49,20 @@ JniCallback::~JniCallback() {
 
   releaseJniEnv(attached_thread);
 }
+
+void JniCallback::catchAndLog(JNIEnv* env) const {
+  jboolean hasException = env->ExceptionCheck();
+  if (hasException == JNI_TRUE) {
+    env->ExceptionDescribe();
```

Comment:
Won't this just write to StdOut/StdErr?

It would be better to use the `Logger` here.

Reply:
not sure how to do this. I saw that ExceptionDescribe was used in other places too. Maybe it should be part of a more global ticket for jni env exception handling